speech/snippets/beta_snippets_test.py

```diff
@@ -31,66 +33,90 @@
 
 
 @Retry()
-def test_transcribe_file_with_enhanced_model(capsys):
-    transcribe_file_with_enhanced_model()
+def test_transcribe_file_with_enhanced_model(
+    capsys: pytest.CaptureFixture
+) -> None:
+    result = transcribe_file_with_enhanced_model()
     out, _ = capsys.readouterr()
 
     assert "Chrome" in out
```

Comment:
My understanding is that part of the clean-up should include redoing the test to use returned result instead of output parsing. After an update `capsys` fixture shouldn't be needed anymore.

Reply:
The above note is applicable to all tests in this file.